genie-agent/src/main/java/com/netflix/genie/agent/execution/services/impl/NaiveFetchingCacheService.java

```diff
@@ -28,7 +28,6 @@
 import org.springframework.context.annotation.Lazy;
```

Comment:
You could go ahead and delete this and its tests. I can't think of any use for it.

Reply:
I was thinking of using it as a fallback. In case we find some env in the future where FileLock has  some issue, we can always resort to using this one was my thought.

Reply:
If filelock doesn't work we probably don't want to revert back to something that does not locking at all.
This is dead code, it makes the build slower and larger so it can go.

Reply:
Deleted